packages/manager-components/src/components/onboarding/onboarding.component.tsx

```diff
@@ -7,7 +7,8 @@ import {
 import { OsdsText } from '@ovhcloud/ods-components/text/react';
 import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
-import placeholderSrc from './placeholder.png';
+
+import placeholderSrc from './assets/placeholder.png';
 
 export type OnboardingLayoutProps = {
```

Comment:
use `React.PropsWithChildren<Props>` to add `children` prop

Reply:
👍